doc/tutorials/rt_linux.rst

```diff
@@ -53,17 +53,50 @@ system on Intel KBL NUC with a SATA SSD as ``/dev/sda`` and an NVME SSD as
 
    a. Download Linux kernel real-time patch::
 
-         $ wget https://mirrors.edge.kernel.org/pub/linux/kernel/projects/rt/4.19/patch-4.19.15-rt12.patch.xz
+         $ wget https://mirrors.edge.kernel.org/pub/linux/kernel/projects/rt/4.19/patch-4.19.31-rt18.patch.xz
 
-   #. Sync the kernel code to acrn-2019w04.5-150000p::
+   #. Sync the kernel code to acrn-2019w17.4-160000p::
 
          $ git clone https://github.com/projectacrn/acrn-kernel.git
-         $ git checkout acrn-2019w04.5-150000p
+         $ git checkout acrn-2019w17.4-160000p
          $ cd acrn-kernel
-         $ xzcat ../patch-4.19.15-rt12.patch.xz | patch -p1
+         $ vim ./mm/swap.c
+         $ xzcat ../patch-4.19.31-rt18.patch.xz | patch -p1
+
+      .. note:: To avoid conflicts please modify swap.c file as follows:
+ 
+      .. code-block:: console
```

Comment:
```suggestion
      .. code-block:: none
```

This probably renders better using `none`, @dbkinder , what do you think?

Reply:
The ``console`` format is a black background with white text, simulating a "console" instead of taking a picture of a console.  So yes,  using ``none`` is better.

That said, your instructions say to use ``vim`` but the contents of this code block look like a patch file.  If you want folks to use a ``patch`` command, give them a patch file and a patch command to use.  If you want them to use ``vim``, give them the text they should enter with ``vim`` and not lines that start with a ``+`` and ``-`` as you're showing below.  (To reduce errors, it would be best to give them a patch file.)